packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -165,6 +166,13 @@ class CupertinoCheckbox extends StatefulWidget {
   /// [RoundedRectangleBorder] with a circular corner radius of 4.0.
   final OutlinedBorder? shape;
 
+  /// The semantic label for the checkbox that will be announced by screen readers.
+  ///
+  /// This is announced in accessibility modes (e.g TalkBack/VoiceOver).
```

Comment:
```suggestion
  /// This is announced by assistive technologies (e.g TalkBack/VoiceOver).
```

Reply:
@chunhtai looks like this came from the description from the same property in the Material checkbox:
https://api.flutter.dev/flutter/material/Checkbox/semanticLabel.html

Should your suggested wording be used in both?

Reply:
I changed the wording for other semanticLabel API docs across the framework.

Reply:
yes, assistive technologies are what we use to address voiceover or talkback. We decided to use this word about 2 to 3 year ago, so there may be some old wording in the code base. I would update them all if found any.